src/cci/cci_query_execute.c

```diff
@@ -3690,7 +3678,7 @@ qe_get_data_int (T_CCI_U_TYPE u_type, char *col_value_p, void *value)
       return CCI_ER_TYPE_CONVERSION;
     }
 
-  *((int *) value) = data;
+  *(int *) value = data;
```

Comment:
What about this style:
at the entry point, add new (int *) variable,
int *i_value = (int *) value;
...
and change this line to:
*((int *) i_value) = data;